### DISCUSSION TITLE
Loitering vs Inertia?

### DISCUSSION THREAD

**Discussion by J-Pipe** [blakeblackshear/frigate#13972] [Ask A Question]
Not sure why I can't find any info on this? What is supposed to be the difference? If I wanted a loitering time of five seconds in a zone, couldn't this already be done by setting an inertia value of fps * 5? I'm seeing the exact same behavior from both. And I would expect that Inertia starts the timer from once the object steps into the zone (restarting each time it steps in again), but it is a cumulative time, just like the loitering time. Not sure what the idea with this was, or if it is just not working correctly.

**Comment by NickM-27** [comment]
Inertia is not cumulative, inertia is inertia-based which means for each frame the object is detected in the zone the score increases by one and for each frame the object is not detected in the zone the score decreases by one. Loitering is cumulative. 

They are similar but exist to serve different purposes / handle different use cases
